opm/material/localad/Evaluation.hpp

```diff
@@ -441,11 +443,27 @@ std::ostream& operator<<(std::ostream& os, const Evaluation<Scalar, VarSetTag, n
 // { std::cout << foo(0) << "\n"; }
 //
 // this will print '0' for bar() and '1' for foobar()...
-//
-#error "Due to some C++ peculiarity regarding function template specialization,"
-#error "the 'evaluation.hh' header must be included before Dune's 'densematrix.hh'!"
+#if !(DUNE_VERSION_NEWER(DUNE_COMMON, 2,4))
+
+#include "Math.hpp"
+
+namespace std {
```

Comment:
It is illegal to add an overload in the std:: namespace. It is also unnecessary due to argument-dependent lookup. It is enough to have abs() in Opm::LocalAd.